src/main/java/frc/trigon/robot/poseestimation/poseestimator/PoseEstimatorConstants.java

```diff
@@ -1,18 +1,16 @@
 package frc.trigon.robot.poseestimation.poseestimator;
 
-import edu.wpi.first.math.VecBuilder;
-import edu.wpi.first.math.Vector;
-import edu.wpi.first.math.numbers.N3;
-
 public class PoseEstimatorConstants {
     public static final double ODOMETRY_FREQUENCY_HERTZ = 250;
 
+    static final double POSE_BUFFER_SIZE_SECONDS = 2;
+
     /**
-     * The vector represents how ambiguous each value of the odometry is.
-     * The first value represents how ambiguous is the x,
-     * the second one for the y, and the third one is for the theta (rotation).
-     * Increase these numbers to trust the estimate less.
+     * Each number represents how ambiguous a value of the odometry is.
+     * The first number represents how ambiguous the x is,
+     * the second one is for the y, and the third one is for the theta (rotation).
+     * The greater these numbers are, the less we trust the estimation.
      */
-    static final Vector<N3> ODOMETRY_AMBIGUITY = VecBuilder.fill(0.003, 0.003, 0.0002);
+    static final double[] AMBIGUITY = {0.003, 0.003, 0.0002};
```

Comment:
This isn't just any ambiguity, this is the odoms std devs
Why not use a vector?

Reply:
A vector represents what it has better. 
Think of it like a pose2d that represents a direction instead.

Reply:
I suppose with good documentation you could use an array.
Maybe even create a record for this?